src/function.rs

```diff
@@ -136,3 +141,42 @@ fn just_executable(_context: &FunctionContext) -> Result<String, String> {
     )
   })
 }
+
+fn file_name(_context: &FunctionContext, file_path: &str) -> Result<String, String> {
+  Utf8Path::new(file_path)
+    .file_name()
+    .map(std::borrow::ToOwned::to_owned)
```

Comment:
You can do `str::to_owned` here, which I think is a little is a little more readable.